buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -817,11 +818,8 @@ class BeamModulePlugin implements Plugin<Project> {
             targetFiles += sourceSet.allJava
           }
           target targetFiles.matching {
-            include '**/*.java'
+            include '**/src/**/*.java'
```

Comment:
What prevents us from doing ```src/**/*.java```?

I believe the path is relative to the build.gradle file applying this plugin.

Reply:
Nothing prevents us from doing that to my knowledge, updated accordingly. Just wanted to keep it less restrictive to be on the safe side. 'src/**..' seems to work for the cases I tested so it makes more sense.